pkgs/development/lua-modules/overrides.nix

```diff
@@ -350,18 +358,16 @@ with prev;
 
     buildInputs = [ pkgs.pkg-config pkgs.libuv ];
 
-    doInstallCheck = true;
-
     # Use system libuv instead of building local and statically linking
     extraVariables = {
       "WITH_SHARED_LIBUV" = "ON";
     };
 
     # we unset the LUA_PATH since the hook erases the interpreter defaults (To fix)
-    installCheckPhase = ''
+    # tests is not run since they are not part of the tarball anymore
+    preCheck = ''
       unset LUA_PATH
       rm tests/test-{dns,thread}.lua
```

Comment:
If tests are not even part of the tarball, why do you still have to remove them?

Reply:
because they may still be helpful when testing from a github checkout (when I test the upgrade for instance).